### ISSUE TITLE
Documentation: Add polished  "how to get help" info

### ISSUE THREAD

**Issue by pushfoo** [pythonarcade/arcade#1298]
## Documentation request:

Thanks to @DragonMoffon for starting up discussion around tutorials and doc again!

### What documentation needs to be added?

A section on how to get help effectively, covering:
- [X] What information to give people when stuck on a problem, sort of like the issue templates we have for GithHub
- [X] How to paste and format code when using discord, including backticks and where to find them on the keyboard
- [X] Links to places and tutorials on more advanced ways to share code, such as pastebin, python discord's pastebin, and github
- [X] Links to github formatting info
- [X] Example screenshots for formatting
- [X] tl;dr example images at the top

### Where should it be added? 

The pages themselves could go in the programming guide, but they could also be placed somewhere directly in the tutorial series.

I would link to this content repeatedly from the tutorial series, and maybe once or twice on the getting started page.

### Why would this be useful?

**tl;dr we frequently have to explain how to use back ticks and share code**

Having a polished version of this documentation readily on hand would make it easier to help people. Highly active communities on discord and elsewhere often have bots or other ways of providing people with 1-2 sentence summaries with links to fuller documentation on this subject. We don't need to go quite that far at the moment, but maybe we could make a single-image infographic that people could send over discord.

**Comment by pushfoo** [comment]
I'm working on this and should have a PR ready this week.
